plugins/blip/basics/components/sitechange/Selector.js

```diff
@@ -0,0 +1,183 @@
+/* 
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2017 Tidepool Project
+ * 
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ * 
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+/* jshint esversion:6 */
+
+var _ = require('lodash');
+var React = require('react');
+var cx = require('classnames');
+
+var basicsActions = require('../../logic/actions');
+var BasicsUtils = require('../BasicsUtils');
+
+var constants = require('../../logic/constants');
+
+var Selector = React.createClass({
+  mixins: [BasicsUtils],
+  propTypes: {
+    data: React.PropTypes.object,
+    selectedSubtotal: React.PropTypes.string.isRequired,
+    selectorOptions: React.PropTypes.object.isRequired,
+    selectorMetaData: React.PropTypes.object.isRequired,
+    updateBasicsSettings: React.PropTypes.func.isRequired,
+    sectionId: React.PropTypes.string.isRequired,
+    trackMetric: React.PropTypes.func.isRequired,
+  },
+  render: function() {
+    var self = this;
+
+    return (
+      <div className="SiteChangeSelector">
+        {this.renderMessage()}
+        {this.renderOptions()}
+      </div>
+    );
+  },
+  renderMessage: function() {
+    if (!this.props.selectorMetaData.hasOwnProperty('latestPump')) {
+      return;
+    }
+
+    var message;
+    var type = this.props.selectedSubtotal;
+    var {
+      latestPump,
+      canUpdateSettings,
+      patientName,
+    } = this.props.selectorMetaData;
+
+    if (canUpdateSettings) {
+      switch(type) {
+        case constants.SITE_CHANGE_TUBING:
+          message = ['We are using ', this.subAction(latestPump, constants.SITE_CHANGE_TUBING), ' to visualize your infusion site changes.'];
+          break;
+        case constants.SITE_CHANGE_CANNULA:
+          message = ['We are using ', this.subAction(latestPump, constants.SITE_CHANGE_CANNULA), ' to visualize your infusion site changes. Choose ', this.subAction(latestPump, constants.SITE_CHANGE_TUBING), ' if those are better indicators.'];
+          break;
+        default:
+          message = ['We do not have enough information to visualize your infusion site changes. Choose ', this.subAction(latestPump, constants.SITE_CHANGE_TUBING), ' or ', this.subAction(latestPump, constants.SITE_CHANGE_CANNULA), ' to indicate your infusion site changes.'];
```

Comment:
👍 This was mostly remedied after reviewing again with product to dramatically simplify copy, but I went ahead and split the array onto different lines, too.